File: Primeiros passos com python/Desafios-de-codigo-em-5-linguagens-diferente/desafioIfs.py
"""
Para ler e escrever dados em Python, utilizamos as seguintes funções:
    - input: lê UMA linha com dado(s) de Entrada do usúario;
    - print: imprime um texto de Saída (Output), pulando linha.
"""

#Função útil para o calculo do imposto (baseado nas aliquotas).
def calcular_imposto(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 0.05
    elif (salario >= 1100.01 and salario <= 2500):
        aliquota = 0.10
    else:
        aliquota = 0.15
    return aliquota * salario

#Lê os valores de Entrada:
valor_salario = float(input())
valor_baneficios = float(input())

#Calcula o imposto através da dunção "calcular_imposto":
valor_imposto = calcular_imposto(valor_salario)
#Calcula e imprime a Saída (com 2 casas decimais):
saida = valor_salario - valor_imposto + valor_baneficios
print(f'{saida:.2f}')

File: Primeiros passos com python/Desafios-de-codigo-em-5-linguagens-diferente/desafioIfs.java
import java.util.Scanner;

public class desafioIfs {

    public static void main(String[] args) {
        //Ler valores de entrada
        Scanner leitorDeEntrada =  new Scanner(System.in);
        float valorSalario = leitorDeEntrada.nextFloat();
        float valorBenefico = leitorDeEntrada.nextFloat();

        float valorImposto = 0;
        
        if(valorSalario >= 0 && valorSalario <= 1100) {
            //atribui a aliquota de 5%
            valorImposto = 0.05F * valorSalario;
        }
        else if(valorSalario >= 110.01 && valorSalario <= 2500) {
            valorImposto = 0.10F * valorSalario;
        }
        else {
            valorImposto = 0.15F * valorSalario;
        }

        float saida = valorSalario - valorImposto + valorBenefico;
        System.out.println(String.format("%.2f", saida));

    }
    
}